app/domain/operations/notices/ivl_enr_notice_trigger.rb

```diff
@@ -49,7 +49,7 @@ def build_addresses(person)
             :address_2 => address.address_2.presence,
             :address_3 => address.address_3.presence,
             :state => address.state,
-            :city => address.city,
+            :city => address.city.titleize,
```

Comment:
From the ticket, it looks like something is changing between the user-entered text of `Test Test` and where it appears in the notice `Test test`. Can we try to discover the root cause of this issue?

If a user enters `Test Test` as their city, that's how it should appear on the notice. We shouldn't need to manipulate the user input to get it to appear correctly.

It seems like we should be removing the code that prints `Test test` rather than forcing it to print the string in Title Case.

Reply:
@markgoho I checked the data from backend how its storing into the document. It stored as user entered. However on notice generation, they want the second word of the city's first character capital letter. then my fix fill work on this.

Reply:
if we're storing it as user entered in the backend, then i expect it to show "Test Test" in the notice when they saved address with city as "Test Test".

So, the problem we're solving here is to always capitalize the second word even when user entered "Test test".

Is that right ?